packages/edit-site/src/store/selectors.js

```diff
@@ -268,32 +324,11 @@ export const getCurrentTemplateTemplateParts = createRegistrySelector(
 			'wp_template_part',
 			{ per_page: -1 }
 		);
-		const templatePartsById = templateParts
-			? // Key template parts by their ID.
-			  templateParts.reduce(
-					( newTemplateParts, part ) => ( {
-						...newTemplateParts,
-						[ part.id ]: part,
-					} ),
-					{}
-			  )
-			: {};
-
-		return ( template.blocks ?? [] )
-			.filter( ( block ) => isTemplatePart( block ) )
-			.map( ( block ) => {
-				const {
-					attributes: { theme, slug },
-				} = block;
-				const templatePartId = `${ theme }//${ slug }`;
-				const templatePart = templatePartsById[ templatePartId ];
-
-				return {
-					templatePart,
-					block,
-				};
-			} )
-			.filter( ( { templatePart } ) => !! templatePart );
+
+		return memoizedGetFilteredTemplatePartBlocks(
+			template.blocks || [],
```

Comment:
Do we really need the `|| []` if we have a default value for the first argument of `getFilteredTemplatePartBlocks()`. OTOH, if we specify the empty array here, why do we need the default value above? One of those seems redundant.

Reply:
Good point, I've switched over to using the `EMPTY_ARRAY` approach you suggested in the other comment.